tests/acceptance/features/apiLocks/lockFiles.feature

```diff
@@ -370,7 +369,7 @@ Feature: lock files
       | spaces           | shared     |
       | spaces           | exclusive  |
 
-  @issue-7638 @issue-7599
```

Comment:
@2403905 Let's not remove the issue tags. it servers as a reference for the future.